recipe/meta.yaml

```diff
@@ -1,26 +1,28 @@
-{% set version = "2.2.0" %}
+{% set version = "1.10.1" %}
 
 package:
   name: conjure-python-client
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/conjure-python-client/conjure-python-client-{{ version }}.tar.gz
-  sha256: f8ea3fb12851173d45c322299c6f761563b3cdb395ac41ae0a0b1cfc6661c584
+  sha256: d68f19a391a949756b68f46c0b4978efa96f782dca701b2059b7a22967b5cb17
 
 build:
-  noarch: python
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.8
+    - python
```

Comment:
These requirements match the 1.10.1 requirements https://github.com/conda-forge/conjure-python-client-feedstock/blob/86a6620f3e80a051d75131d6e258891381d27da5/recipe/meta.yaml